src/applications/vre/components/GetFormHelp.jsx

```diff
@@ -1,25 +1,17 @@
 import React from 'react';
 import Telephone, {
   CONTACTS,
+  PATTERNS,
 } from '@department-of-veterans-affairs/formation-react/Telephone';
 
-export default function GetFormHelp() {
-  return (
-    <div>
-      <p className="help-talk">Enrollment or Eligibility questions:</p>
-      <p className="help-phone-number">
-        <a className="help-phone-number-link" href="tel:1-877-222-8387">
-          877-222-8387
-        </a>
-        <br />
-        TTY:{' '}
-        <Telephone
-          className="help-phone-number-link"
-          contact={CONTACTS.HELP_TTY}
-        />
-        <br />
-        Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. ET
-      </p>
-    </div>
-  );
-}
+const GetFormHelp = () => (
+  <p className="help-talk">
+    Need help filling out the form or have questions about eligibility? Please
+    call VA Benefits and Services at{' '}
+    <Telephone contact={CONTACTS.VA_BENEFITS} />.<br />
+    If you have hearing loss, call TTY:{' '}
+    <Telephone contact={CONTACTS['711']} pattern={PATTERNS['911']} />.
```

Comment:
Pattern is `3_DIGIT` now

https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation-react/src/components/Telephone/Telephone.jsx#L34